modules/dxp/apps/portal-search-tuning/portal-search-tuning-synonyms-web-test/src/testIntegration/java/com/liferay/portal/search/tuning/synonyms/web/test/SynonymSearchTest.java

```diff
@@ -251,7 +257,7 @@ protected static Dictionary<String, Object> setUpSynonymsProperties() {
 			"filterNames",
 			new String[] {
 				"liferay_filter_synonym_en", "liferay_filter_synonym_es",
-				"custom-synonym-filter-fr"
```

Comment:
maybe we can have each test for each language setting the only synonym enabled the one for the language being tested, and later we can add tests that allow multiple synonyms enabled.